tests/e2e/specs/productDetailPage_spec.js

```diff
@@ -0,0 +1,46 @@
+describe('Product detail page', () => {
+  before(() => {
+    cy.visit('/product/lemare-booties-0778-grey/M0E20000000E0WX');
+  });
+
+  it('Displays a single product details', () => {
+    cy.get('[data-test=product-description]', { timeout: 20000 })
+      .then(($product) => {
+        cy.wrap($product)
+          .find('[data-test=pdp-product-title]')
+          .should('contain', 'Booties Lemare grey');
+
+        cy.wrap($product)
+          .find('[data-test=product-sku]')
+          .should('contain', 'M0E20000000E0WX');
+
+        cy.wrap($product)
+          .find('[data-test=product-discounted-price]')
+          .should('contain', '248,75 €');
+        cy.wrap($product)
+          .find('[data-test=product-discount-price]')
+          .should('contain', '174,12 €');
+
+        cy.wrap($product)
+          .find('[data-test=panel-default]')
+          .should('exist')
+          .find('[data-test=accordion-toggle]')
```

Comment:
For the `data-test`, try to write what you feel that the `div` contains, don't base your name on the CSS styles that you see, because they are generally bad.
So given that, I hope you would consider naming these both different than `panel-default` and `accordion-toggle`.